### ISSUE TITLE
France.tv works well except this URL can't find video formats

### ISSUE THREAD

**Issue by TexasDad** [ytdl-org/youtube-dl#22426]
## Checklist

- [x] I'm reporting a broken site support
- [x] I've verified that I'm running youtube-dl version **2019.09.12.1**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar issues including closed ones

## Verbose log

 
```
PASTE VERBOSE LOG HERE

youtube-dl -F https://www.france.tv/documentaires/art-culture/1062093-regard-sur-les-objets-polynesiens.html

[FranceTVSite] 1062093-regard-sur-les-objets-polynesiens: Downloading webpage
[FranceTV] 49749350-f554-4714-97e1-319bfe8e3d72: Downloading video JSON
[FranceTV] 49749350-f554-4714-97e1-319bfe8e3d72: Downloading geo restriction info
ERROR: No video formats found; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
[telematin@workman Artes]$ youtube-dl --verbose -F https://www.france.tv/documentaires/art-culture/1062093-regard-sur-les-objets-polynesiens.html
[debug] System config: [u'--prefer-free-formats']
[debug] User config: [u'--ignore-errors', u'--restrict-filenames']
[debug] Custom config: []
[debug] Command-line args: [u'--verbose', u'-F', u'https://www.france.tv/documentaires/art-culture/1062093-regard-sur-les-objets-polynesiens.html']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2019.09.12.1
[debug] Python version 2.7.15 (CPython) - Linux-5.0.17-200.fc29.x86_64-x86_64-with-fedora-29-Twenty_Nine
[debug] exe versions: ffmpeg 4.0.4, ffprobe 4.0.4
[debug] Proxy map: {}
[FranceTVSite] 1062093-regard-sur-les-objets-polynesiens: Downloading webpage
[FranceTV] 49749350-f554-4714-97e1-319bfe8e3d72: Downloading video JSON
[FranceTV] 49749350-f554-4714-97e1-319bfe8e3d72: Downloading geo restriction info
ERROR: No video formats found; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/usr/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 796, in extract_info
    ie_result = ie.extract(url)
  File "/usr/bin/youtube-dl/youtube_dl/extractor/common.py", line 530, in extract
    ie_result = self._real_extract(url)
  File "/usr/bin/youtube-dl/youtube_dl/extractor/francetv.py", line 209, in _real_extract
    return self._extract_video(video_id, catalog)
  File "/usr/bin/youtube-dl/youtube_dl/extractor/francetv.py", line 169, in _extract_video
    self._sort_formats(formats)
  File "/usr/bin/youtube-dl/youtube_dl/extractor/common.py", line 1327, in _sort_formats
    raise ExtractorError('No video formats found')
ExtractorError: No video formats found; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.


```

## Description

 
I went to download a video off of the France 0 channel only to discover that YT couldn't find the video formats. 

YT works perfectly for channels France 2, France 3, France 4, and France 5.

My typical command on the channels is this ...

youtube-dl -k -f best --write-sub --embed-subs --restrict-filenames

Often, when I am unsure of the video formats, I will run 

youtube-dl -F before I do anything else. In this case, there appears to be no video list.

I noticed there is little difference in the URL for this video, except for one minor detail. Typically, for France 2, you will see http://france.tv/france-2/ in the final URL. 

But, in this case, for France 0, I see that there is no channel sub domain listed for the channel and it goes directly to the documentaries sub domain. It ought to make no difference at all to the page since it seems to be configure to show the videos in the same way. But, maybe it is.

Anyway, I searched the bugtracker and can't seem to find another incident of this problem, so I decided to post a bug per the recommendation of the program.

Thank you in advance for the awesome program,

Texas Dad

**Comment by TexasDad** [comment]
Oh, btw, I am running behind a VPN which is geo-located in France.

**Comment by Europe57** [comment]
Ok in France, tested.

```
youtube-dl-F https://www.france.tv/documentaires/art-culture/1062093-regard-sur-les-objets-polynesiens.html
[FranceTVSite] 1062093-regard-sur-les-objets-polynesiens: Downloading webpage
[FranceTV] 49749350-f554-4714-97e1-319bfe8e3d72: Downloading video JSON
[FranceTV] 49749350-f554-4714-97e1-319bfe8e3d72: Downloading geo restriction info
[FranceTV] 49749350-f554-4714-97e1-319bfe8e3d72: Downloading signed hls_v5_os manifest URL
[FranceTV] 49749350-f554-4714-97e1-319bfe8e3d72: Downloading m3u8 information
[info] Available formats for 49749350-f554-4714-97e1-319bfe8e3d72:
format code                             extension  resolution note
hls_v5_os-audio-aacl-64-Audio_Français  mp4        audio only [fr]
hls_v5_os-191                           mp4        256x144     191k , avc1.42C01E, 25.0fps, video only
hls_v5_os-321                           mp4        320x180     321k , avc1.42C01E, 25.0fps, video only
hls_v5_os-609                           mp4        512x288     609k , avc1.42C01E, 25.0fps, video only
hls_v5_os-880                           mp4        704x396     880k , avc1.4D401F, 25.0fps, video only
hls_v5_os-1554                          mp4        1024x576   1554k , avc1.4D401F, 25.0fps, video only (best)
```

**Comment by TexasDad** [comment]
Thank you. Or, should I say, Merci. 

I just tested it and it worked too. Weird. 

The only thing I can imagine created the problem then is an issue with the France.tv server.

I have found from past experience on the France.tv pages that they love to change their content frequently. It could well be that I dropped in on a page they were testing and I didn't know it at the time. 

Which means that in the future, this issue will not only pop up again, but it will affect the whole range of pages France.tv is populating.
